schema/elasticsearch/visibility/versioned/v0/index_template_v6.json

```diff
@@ -5,8 +5,9 @@
   ],
   "settings": {
     "index": {
-      "number_of_shards": "5",
-      "number_of_replicas": "0"
+      "number_of_shards": "1",
+      "number_of_replicas": "0",
+      "auto_expand_replicas": "0-2"
```

Comment:
Why `0-2` but not `0-5`, for example?

Reply:
Default replication factor is 1, but the general number of shards depends on the applying templates on the index as well, so this may be altered.

How many replicas? Depends on the cluster size and how much you care about your data, and how plausible it is that you loose the node.

Usually 2 replicas are enough - you can survive switching one node to maintenance (one replica out) and in the same time loose one extra node (second replica out by disaster) and still have primary shard.
For small clusters like 3-5 nodes then 1 or 2 replicas are enough, for bigger clusters it's worth to bump it to 3, but IMHO 5 would be excessive unless you have a lot of reads

Auto expand replicas helps to scale-in/scale-out shards in case of adding/removing nodes if elasticsearch master controller decides nodes are in different failure domains etc, and thus removes the need to manually alter existing templates.